### PR TITLE
small design modifications

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: src/lib/components/carrousel.svelte [hunk 1]
```diff
@@ -23,47 +23,44 @@
...
+<div class="carrousel">
+  {#if images.length}
+    {#each [images[index]] as { id, path } (index)}
+      <div class="image">
+        <img transition:fade|local sizes="80vw" srcset="{path}, 1200w" {id} src={path} alt={id} />
+      </div>
+    {/each}
+  {/if}
+  <slot {index} />
+  {#if images.length > 0}
```
Comment (zwergius) [inline src/lib/components/carrousel.svelte:35]: duplicated if

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -18,8 +27,16 @@
...
+    color: var(--interactive-button);
+  }
+  .social-media {
+    gap: 10px;
+    display: flex;
+  }
+
   h3 {
-    font-size: var(--25px);
+    font-size: 25px;
```
Comment (zwergius) [inline src/lib/components/footer.svelte:39]: I liked var(--25px) better

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/carrousel.svelte (hunk 1 above)
Comment (zwergius) [reply src/lib/components/carrousel.svelte:34]: no need for 0 check

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/list-store/+page.svelte [hunk 3]
```diff
@@ -38,7 +38,7 @@
     <ul>
       {#each stores as { name, href }}
         <li>
-          <Links {href}>{name} &#8594;</Links>
+          <Links {href}><span class="truncate"> {name} </span> &#8594;</Links>
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:41]: what is it with the extra spaces ?

File: src/routes/list-store/+page.svelte [hunk 4]
```diff
@@ -47,6 +47,13 @@
...
 <style>
   .container :global(a) {
+    font-size: var(--50px);
+    justify-content: center;
+    align-items: center;
+  }
+
+  .truncate {
+    display: block;
     font-size: var(--50px);
```
Comment (zwergius) [inline src/routes/list-store/+page.svelte:57]: you don't need font-size here

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/+layout.svelte [hunk 5]
```diff
@@ -20,19 +20,22 @@
...
 </div>
 
 <Footer />
 
 <style>
-  .box {
+  .wraper {
     display: flex;
     flex: 1;
+    margin-top: 280px;
```
Comment (zwergius) [inline src/routes/+layout.svelte:33]: no

File: src/routes/+layout.svelte [hunk 6]
```diff
@@ -49,6 +52,10 @@
     display: none;
   }
   @media only screen and (max-width: 1024px) {
+    .wraper {
+      margin-top: 350px;
```
Comment (zwergius) [inline src/routes/+layout.svelte:56]: no

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/list-store/+page.svelte (hunk 3 above)
Comment (Javitocatral) [reply src/routes/list-store/+page.svelte]: I like that extra space, I know you don't haha

**Review by zwergius — APPROVED** [review]
LGTM
ok usa

**Pull request merged**